Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -7,380 +7,329 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		090598F5C53313C2A4A07FB878003AA9 /* Pods-Pelican_Tests-umbrella.h in Headers */ = {isa = PBXBuildFile; fileRef = CFE4467604881B562C155DC652804E16 /* Pods-Pelican_Tests-umbrella.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		200D5BC0741ED20BDC67EDF92DCD960F /* Pods-Pelican_Tests-dummy.m in Sources */ = {isa = PBXBuildFile; fileRef = 92A7BA74BF7CA28F32F68CB8CEFD3D42 /* Pods-Pelican_Tests-dummy.m */; };
-		47325C07619058C15A06BECC5F37CAE2 /* Pelican-dummy.m in Sources */ = {isa = PBXBuildFile; fileRef = DD40E5E2549FDBBB2451BE1991666AD6 /* Pelican-dummy.m */; };
-		50A10044A31A1F890E210A9FDEA0999B /* Pelican-umbrella.h in Headers */ = {isa = PBXBuildFile; fileRef = F5A3F31FF86BDA12B88363D7FF82E7E9 /* Pelican-umbrella.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		591E709CA8644E662B57791678B3D2AD /* GroupedTasks.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5AB71903B78C8AACD5F2C6FF40964BB4 /* GroupedTasks.swift */; };
-		8158933D8B2B2F69A6A21135654A4717 /* Foundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5A16F4CFC63FAC439D7A04994F579A03 /* Foundation.framework */; };
-		95FB10B00EBD7232EB1CEBF175CFC8E6 /* PelicanStorage.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5FB77F3BBF22321BECDD1AECAC8281F4 /* PelicanStorage.swift */; };
-		9D01A2C8A536FB1C6387063D70899D9D /* Pelican.swift in Sources */ = {isa = PBXBuildFile; fileRef = FA504DCBAD62DB3FDA146A84283A449B /* Pelican.swift */; };
-		AAD1C9D4960F10127CBE23791515BADD /* Foundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5A16F4CFC63FAC439D7A04994F579A03 /* Foundation.framework */; };
-		AF59818D223B19B2003D1648 /* TaskContainer.swift in Sources */ = {isa = PBXBuildFile; fileRef = AF59818C223B19B2003D1648 /* TaskContainer.swift */; };
-		BFA9656243ABCCF3E3E7963B2709EFAF /* StorablePelicanTask.swift in Sources */ = {isa = PBXBuildFile; fileRef = 82D2BC5DF4C13B77475EE33C212D14B0 /* StorablePelicanTask.swift */; };
-		E041732AFCFFCA64B27CA4AA0EEA8A1C /* PelicanUserDefaultsStorage.swift in Sources */ = {isa = PBXBuildFile; fileRef = CAD38F0154156D979824124EEE997040 /* PelicanUserDefaultsStorage.swift */; };
+		0AD1111D98AC406EA61E24E31DFABE82 /* Pods-Pelican_Tests-umbrella.h in Headers */ = {isa = PBXBuildFile; fileRef = D07DBF3A0945B3E93EF8DDCF3C515373 /* Pods-Pelican_Tests-umbrella.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		331C93F1E1D5435E341E7836F7352119 /* Pelican-umbrella.h in Headers */ = {isa = PBXBuildFile; fileRef = 71DADF09EF7446519C5A1B22DA629C2D /* Pelican-umbrella.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
Are all of these project changes generated by the new version of CocoaPods?

Reply:
Yep